src/realm.js

```diff
@@ -232,6 +232,7 @@ export class Realm {
       metadata?: any
     ) => [Completion | Reference | Value, BabelNode, Array<BabelNodeStatement>],
   };
+  simplifyAbstractValue: (AbstractValue, boolean) => Value;
```

Comment:
Mysterious boolean.
Is this function meant to be path sensitive?

Reply:
I'll document or remove.